src/components/ToolIcon.scss

```diff
@@ -33,6 +33,10 @@
 
   & + .ToolIcon__label {
     margin-left: 0;
+    &.rtl {
+      margin-right: 0;
+      margin-left: 0.8em;
+    }
```

Comment:
```suggestion
    margin-inline-start: 0;
```

Reply:
(we should audit the codebase for uses of `left` and `right` and make sure they’re either switched to the semantic ones like `start` and `end` or properly handled in rtl)